src/main/java/racing/web/controller/RacingController.java

```diff
@@ -26,34 +29,45 @@ public RacingController(RacingGameService racingGameService) {
     }
 
     @PostMapping("/plays")
-    public ResponseEntity<RacingGameResultResponse> playGame(@RequestBody RacingGameInfoRequest request) {
+    public ResponseEntity<RacingGameSaveResponse> playGame(@RequestBody RacingGameInfoRequest request) {
         RequestValidator<RacingGameInfoRequest> requestValidator = new RacingGameInfoRequestValidator();
         requestValidator.validate(request);
 
-        Cars cars = CarFactory.carFactory(request.getNames());
-        Long gameId = racingGameService.saveGameByCount(request.getCount());
-        racingGameService.playGame(gameId, cars);
+        Long gameId = racingGameService.playNewGame(request.getCount(), request.getNames());
 
-        return ResponseEntity.ok(getRacingGameResultResponse(cars));
+        return ResponseEntity
+                .status(HttpStatus.CREATED)
+                .body(new RacingGameSaveResponse(gameId));
+    }
+
+    @GetMapping("/result/{gameId}")
+    public ResponseEntity<RacingGameResultResponse> getSingleGameResult(
+            @PathVariable(value = "gameId") String gameIdRequest) {
+        Long gameId = Long.valueOf(gameIdRequest);
```

Comment:
사소하지만 바로 Long타입으로 받아서 캐스팅하지 않을 수도 있습니다!

```suggestion
    public ResponseEntity<RacingGameResultResponse> getSingleGameResult(
            @PathVariable(value = "gameId") Long gameIdRequest) {
```